.gitlab-ci.yml

```diff
@@ -55,191 +63,171 @@
     variables:
       - $SIMP_FULL_MATRIX
 
-stages:
-  - validation
-  - unit
-  - acceptance
-  - deploy
-
-# Puppet 4.8 for SIMP 6.0 + 6.1 support
+# Puppet + testing environments
 # --------------------------------------
-pup4_8-validation:
-  stage: validation
-  tags:
-    - docker
-  image: ruby:2.1
+.pup_prev: &pup_prev
```

Comment:
Trying to convert these YAML anchors and job names to something generic and versionless like `_current`/`_prev` (instead of the Puppet versions they actually set) is not a good idea for lots of reasons—see discussion in [pupmod-simp-ima#4](https://github.com/simp/pupmod-simp-ima/pull/4#pullrequestreview-170929349)